client/src/components/search/searchBar/search-bar-footer.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { searchPageUrl } from '../../../utils/algolia-locale-setup';
+import NoHitsSuggestion from './no-hits-suggestion';
+
+interface SearchBarFooterProps {
+  hasHits: boolean;
+  query?: string;
+  handleMouseEnter: (e: React.SyntheticEvent<HTMLElement, Event>) => void;
+  handleMouseLeave: (e: React.SyntheticEvent<HTMLElement, Event>) => void;
+}
+
+const SearchBarFooter = ({
+  hasHits,
+  query,
+  handleMouseEnter,
+  handleMouseLeave
+}: SearchBarFooterProps) => {
+  const { t } = useTranslation();
+  const title = t('search.no-tutorials');
+
+  const searchUrl = searchPageUrl;
+
+  if (hasHits && query) {
```

Comment:
Since `!query` means we return `null`, you can just do that first and simplify the logic to `if (hasHits)`. That or `return hasHits ? ... : ...`